README.md

```diff
@@ -31,6 +31,10 @@ In the `env.config.js` file, add the following
 
 The DatadogLoggingService is a concrete implementation of the logging service interface that sends messages to Datadog that can be seen in Datadog RUM and Datadog browser logs. When in development mode, all messages will instead be sent to the console.
 
+When you configure a Datadog logging implementation, you'll set a number of variables used by the DatadogLoggingService. Most (eg. `DATADOG_APPLICATION_ID`, `DATADOG_SESSION_SAMPLE_RATE`) are either supplied to you by Datadog when you set up a RUM application, or are options about how much data we should be saving. Optional variables include:
```

Comment:
nit: Is it worth explicitly listing out all the possible environment variable config settings here, which are required/optional, etc. so consumers don't necessarily need to read the code implementation to understand how to configure the MFE?

Reply:
I did that at first, and then I second-guessed myself, so I'll take your nit as a gut-check on that and go back to my original.